builder/index.html

```diff
@@ -1275,20 +1275,7 @@
                     </div>
                     <div id="all-spells-display" class="row row-cols-1 gy-3 text-center scaled-font pe-0">
                         <div class = "col pe-0">
-                            <div class = "col spell-display spell-expand dark-5 rounded dark-shadow pt-2 border border-dark" id="spell0-infoAvg">spell1</div>
-                            <div class = "col spell-display dark-5 rounded dark-shadow py-2" id = "spell0-info" style="display: none;">Spell 1</div>
-                        </div>
-                        <div class = "col pe-0">
-                            <div class = "col spell-display spell-expand dark-5 rounded dark-shadow pt-2 border border-dark" id="spell1-infoAvg">spell2</div>
-                            <div class = "col spell-display dark-5 rounded dark-shadow py-2" id = "spell1-info" style="display: none;">Spell 2</div>
-                        </div>
-                        <div class = "col pe-0">
-                            <div class = "col spell-display spell-expand dark-5 rounded dark-shadow pt-2 border border-dark" id="spell2-infoAvg">spell3</div>
-                            <div class = "col spell-display dark-5 rounded dark-shadow py-2" id = "spell2-info" style="display: none;">Spell 3</div>
-                        </div>
-                        <div class = "col pe-0">
-                            <div class = "col spell-display spell-expand dark-5 rounded dark-shadow pt-2 border border-dark" id="spell3-infoAvg">spell4</div>
-                            <div class = "col spell-display dark-5 rounded dark-shadow py-2" id = "spell3-info" style="display: none;">Spell 4</div>
+                            <div class = "col spell-display spell-expand dark-5 rounded dark-shadow pt-2 border border-dark" id="spell0-infoAvg">Input a weapon to see abilities!</div>
```

Comment:
idk if this is how the system works rn but using spell0-infoAvg as an id might get confused later in the system for an actual 0th (1st in array) spell or smth

i assume that this div and its container (col pe-0) get deleted too when a weapon gets input

Reply:
yeah, all gets deleted before anything is updated

todo: make melee display better...